vj4/ui/templates/problem_settings.html

```diff
@@ -23,6 +23,7 @@ <h1 class="section__title">{{ _('Settings') }}</h1>
           {{ form.form_checkbox(label='Hidden', help_text='Note: Problem title may not be hidden.', name='hidden', value=pdoc['hidden']) }}
           {{ form.form_text(columns=12, label='Category', help_text='Format: category 1, sub category 1.1, sub category 1.2, ..., sub category 1.x, ..., category n, sub category n.1, sub category n.2, ..., sub category n.m, ...', name='category', value=pdoc['category']|join(', ')) }}
           {{ form.form_text(columns=12, label='Tags', help_text='Splitted by \', \'.', name='tag', value=pdoc['tag']|join(', ')) }}
+          {{ form.form_text(columns=12, label='ACCode', help_text='', name='accode', value=(pdoc['accode'] or '' )) }}
```

Comment:
`help_text='This message will be presented to those whose submissions are accepted.'` and the translation.